src/Xrefcheck/Verify.hs

```diff
@@ -511,88 +512,51 @@ verifyReference
       VerifyResult [ExternalHttpTooManyRequests retryAfter] -> retryAfter
       _ -> Nothing
 
-    isVirtual = matchesGlobPatterns root (ecIgnoreLocalRefsTo cExclusions)
+    isVirtual canonicalRoot = matchesGlobPatterns canonicalRoot (ecIgnoreLocalRefsTo cExclusions)
+
+    checkRef mAnchor canonicalRoot referredFile relativeToRoot = verifying $
```

Comment:
Could you document what exactly the `relativeToRoot` argument stands for?

Reply:
It is quite embarrassing, because its only purpose is to keep track of the link path relative to the repository root, before canonicalizing it, so it can be used in the error report if the link points out of the repository. Maybe this can be done in a neater way?

Reply:
Thanks :+1: 

Maybe call this argument like `shownFilepath`, since in `RIFileLocal` case it is not relative to the root?

While `relativeToRoot` naming in `case rInfo of` seems fine, it exactly indicates the meaning of the path it is being assigned.